### DISCUSSION TITLE
Investments in increased connection capacity with PTDF and LODF physics

### DISCUSSION THREAD

**Discussion by DillonJ** [spine-tools/SpineOpt.jl#868] [Ideas]
Currently, when using PTDF or LODF physics and connection investments, we attempt to capture the impact of the investment on the network characteristics using the line outage distribution factor (LODF). The mutual impact of multiple investments is not considered so this approach is useful when considering multiple alternative mutually exclusive investments or when we can assume investments candidates have limited inpact on each other. We can improve on this but that is another issue.

Here, we would like to make the capacity of existing connections an investment variable without considering the impact on network characteristics. This is a much simpler problem and is pretty easy to do when not using PTDF or LODF physics - we simply have `initial_connections_invested_available` set so that we have the right starting capacity. However when using PTDF or LODF physics we can't do this without invoking the constraints and code that consider the impact on the network characteristics.

Hopefully this is a relatively minor change - but it needs careful thinking through.

**Comment by manuelma** [comment]
So if I understand correctly the challenge here is to allow investments in connection capacity as an alternative to investments in the 'number' of connections?

In other words, we want to support both options via an additional parameter in the DB (or an alternative value for an existing parameter?)

@DillonJ thoughts?

**Comment by manuelma** [comment]
Done via https://github.com/spine-tools/SpineOpt.jl/pull/951
